src/transformers/models/clip/modeling_tf_clip.py

```diff
@@ -766,7 +877,21 @@ def build(self, input_shape: tf.TensorShape = None):
             name="logit_scale",
         )
 
-        super().build(input_shape)
+        if self.built:
```

Comment:
Same q here. I'm not going to message any time I see it in case it's nothing but if it's not then you'll have to go through the PR to find all the cases!